### PR TITLE
Ex. 11 Lemonade Sales

### PR DESCRIPTION
**Pull request by KinkyHairCoder**

### CODE REVIEW

File: Assignments/stu1/data-modeling.sql [hunk 1]
```diff
@@ -1,7 +1,97 @@
 /**
  * Ex. 11
  */
+BEGIN;
```
Comment (mburolla) [inline Assignments/stu1/data-modeling.sql:4]: Did you run this script on your database?  I get the following error:
```
ERROR:  there is no unique constraint matching given keys for referenced table "Lemonade Stand"
SQL state: 42830
```

This is a similar error that Diane got.  Maybe the one to many relationship is backwards?

Comment (KinkyHairCoder) [reply Assignments/stu1/data-modeling.sql:4]: I'll try it. I messed up anyway and I noticed once she shared her screen.

File: Assignments/stu1/data-modeling.sql [hunk 2]
```diff
@@ -3,90 +3,70 @@
...
-
-
-ALTER TABLE IF EXISTS public."Lemonade sizes"
-    ADD CONSTRAINT "$2.00" FOREIGN KEY (lemonade_prices)
-    REFERENCES public."Lemonade sizes" (lemonade_prices) MATCH SIMPLE
-    ON UPDATE NO ACTION
-    ON DELETE NO ACTION
-    NOT VALID;
+CREATE SCHEMA logging;
+CREATE TABLE logging.t_history (
```
Comment (mburolla) [inline Assignments/stu1/data-modeling.sql]: What is all this stuff?  Should we close this PR and start over?

Comment (KinkyHairCoder) [reply Assignments/stu1/data-modeling.sql]: I was trying to create something to track the sales lol.

File: Assignments/stu1/data-modeling.sql [hunk 3]
```diff
@@ -1,7 +1,44 @@
...
+insert into size (lemonade_size, lemonade_price) values ('Small', '$1.00');
+insert into size (lemonade_size, lemonade_price) values ('Medium', '$2.00');
+insert into size (lemonade_size, lemonade_price) values ('Large', '$3.00');
+
+insert into lemonade (lemonade_flavor) values ('Lemon');
+insert into lemonade (lemonade_flavor) values ('Strawberry');
+
+select count(*) from cat;
+
+CREATE TABLE IF NOT EXISTS public.lemonade
```
Comment (mburolla) [inline Assignments/stu1/data-modeling.sql:22]: I think we need a day or date field on this table.  I'm not sure how this model supports keeping track of daily lemonade sales.

Comment (mburolla) [reply Assignments/stu1/data-modeling.sql:22]: Unless I'm missing something...

Comment (KinkyHairCoder) [reply Assignments/stu1/data-modeling.sql:22]: I had this select count(lemonade_size_id) from lemonade; but I forgot to change it. I am having a hard time trying to get it to work.

**Comment by mburolla** [comment]
Here's an idea...you could create an Excel spreadsheet that would keep track of daily lemonade sales (price & flavor), and maybe convert that to a relational data model.

**Pull request merged**